lib/Config.js

```diff
@@ -54,6 +54,59 @@ function restEndpointsAssert(restEndpoints, locationConstraints) {
         'bad config: rest endpoint target not in locationConstraints');
 }
 
+function gcpLocationConstraintAssert(location, locationObj) {
+    const {
+        gcpEndpoint,
+        bucketName,
+        mpuBucketName,
+        overflowBucketName,
+        serviceCredentials,
+    } = locationObj.details;
+    const defaultKeyFileFromEnv =
+        process.env.GOOGLE_APPLICATION_CREDENTIALS;
```

Comment:
We could not use these environment variables if we bootstrap the app with multiple GCP locations over different accounts.

Reply:
I see. Will remove the default keyfile environment